ci_framework/playbooks/01-bootstrap.yml

```diff
@@ -19,7 +19,7 @@
           {{
             hostvars[inventory_hostname] |
             dict2items |
-            selectattr("key", "match", "^(cifmw|pre|post)_(?!install_yamls|openshift).*") |
+            selectattr("key", "match", "^(cifmw|pre|post)_(?!install_yamls|openshift_token|openshift_login).*") |
```

Comment:
probably the `_password` as well as `_user` as well ?

```YAML
cifmw_openshift_api: api.crc.testing:6443
cifmw_openshift_kubeconfig: /home/zuul/.crc/machines/crc/kubeconfig
cifmw_openshift_password: '123456789'
cifmw_openshift_setup_skip_internal_registry: true
cifmw_openshift_setup_skip_internal_registry_tls_verify: true
cifmw_openshift_skip_tls_verify: true
cifmw_openshift_user: kubeadmin
```

Reply:
I skip those ones on purpose, cause both fields are user provided, and if provided means we are login in based on user/password, same value as the kubeconfig, that may not be provided. I'd keep them.

Reply:
lgtm then.